### DISCUSSION TITLE
Is this program compatible with the PS3 Blu-Ray Remotes?

### DISCUSSION THREAD

**Discussion by SecularSteve** [nefarius/BthPS3#31] [Q&A]
![maxresdefault](https://user-images.githubusercontent.com/33793273/137335603-5b4ced57-b5c8-40e1-904b-de15d7fe6181.jpg)

I own one myself, and I can't connect them to my PC, even though my controller is working with no issues wirelessly.
Before you ask, **yes**, my batteries are full...

Do these Remote Controls run on a "separate wavelength", or were they just never considered for this program?

**Comment by nefarius** [comment]
No. Classic TV and console remotes typically communicate via infrared, entirely different technology, protocol etc. so not a topic for these drivers or Bluetooth in general I'm afraid.

**Reply by SecularSteve** [reply]
You do realise that these remotes (more specifically, the top one) don't communicate with IR, but they predominantly use Bluetooth to communicate with the PS3 instead, right?

![photo03](https://user-images.githubusercontent.com/33793273/137355617-189f786a-4690-41dd-a333-e7b3d0ab6c60.jpg)

I can understand in the case of the bottom one, but is the top one just simply incapable of doing it because of some other unrelated issues?

It's not a problem, I'd just like a better clarification...

**Reply by nefarius** [reply]
No, I do not. It was an assumption as I never owned one of these, or dissected or in general cared for any TV remotes 😛 

BthPS3 was made for the **game controllers** clearly outlined in the docs, nothing more, nothing less. It's that simple.
